apps/docs/pages/guides/cli/config.tsx

```diff
@@ -39,7 +39,7 @@ export default function Config() {
           </div>
         </div>
         <div className="md:col-span-3">
-          <div className="sticky top-20 border-l">
+          <div className="sticky overflow-auto max-h-[calc(100vh-60px-5rem)] pb-3 top-20 border-l">
```

Comment:
Sorry, last request since we're here: can you crank the `top` value to `top-32`?
That way, the side navigator stays in place when you scroll. (Like in other pages)

Thanks!

Reply:
Yeah, no problem